### PR TITLE
Add some tests to envoy

### PR DESCRIPTION
**Pull request by JulienTant**

### CODE REVIEW

**Comment by JulienTant** [comment]
Ok it looks like my previous PR is here too... I let the two open, and will remove one if needed.

File: composer.json [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
-	"require": {
-		"illuminate/support": "~4.1|~5.0",
-		"nategood/httpful": "~0.2",
-		"symfony/console": "~2.3",
-		"symfony/process": "~2.3"
-	},
-	"bin": [
-		"envoy"
-	]
+  "name": "laravel/envoy",
```
Comment (GrahamCampbell) [inline composer.json]: Please revert the indentation changes in this file.

File: tests/TaskContainerTest.php [hunk 2]
```diff
@@ -0,0 +1,200 @@
+<?php
+namespace Laravel\Envoy\Tests;
```
Comment (GrahamCampbell) [inline tests/TaskContainerTest.php]: This needs to go on the same line as the opening `<?php`.

File: tests/TaskContainerTest.php [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    private function assertMacro($value)
+    {
+        $this->assertInternalType('array', $value, 'The given value is not a Macro');
+    }
+
+    private function assertTask($value)
+    {
+        $this->assertInstanceOf('\Laravel\Envoy\Task', $value, 'The given value is not a Task');
+    }
+}
```
Comment (GrahamCampbell) [inline tests/TaskContainerTest.php]: all files must end in a single new line character.

**Comment by taylorotwell** [comment]
Has quite a bit of formatting problems.

**Pull request closed without merging**